pkg/csi/service/common/constants.go

```diff
@@ -162,16 +162,24 @@ const (
 	// IPs is Client IP address, IP range or IP subnet
 	IPs string = "ips"
 
-	// CSINamespace is the namespace of pvCSI in TKC Cluster
-	CSINamespace = "vmware-system-csi"
+	// CSINamespaceVanillaK8S is the namespace of CSI driver in Vanilla K8S
+	CSINamespaceVanillaK8S = "kube-system"
+
+	// CSINamespaceSVCluster is the namespace of CSI in Supervisor
+	CSINamespaceSVCluster = "vmware-system-csi"
+
+	// CSINamespaceTkgCluster is the namespace of CSI in TKG Cluster
+	CSINamespaceTkgCluster = "vmware-system-csi"
 
 	// CSIFeatureStatesConfigMapName is the name of configmap to store FSS values
 	CSIFeatureStatesConfigMapName = "csi-feature-states"
 )
 
 // Supported container orchestrators
 const (
-	Kubernetes = iota // Default container orchestor for TKC, Supervisor Cluster and Vanilla K8s
+	VanillaKubernetes = iota // Default container orchestor for TKC, Supervisor Cluster and Vanilla K8s
+	Workload
```

Comment:
Let's not add Workload and TkgCluster as Container orchestrators. They are flavors.
Container orchestrators are Kubernetes, Docker EE, Marathon Mesos etc..

Reply:
Done